perfume-api/src/main/java/io/perfume/api/sample/infrastructure/api/SampleController.java

```diff
@@ -0,0 +1,60 @@
+package io.perfume.api.sample.infrastructure.api;
+
+
+import io.perfume.api.sample.application.SampleService;
+import io.perfume.api.sample.application.dto.SampleResult;
+import io.perfume.api.sample.infrastructure.api.dto.CreateSampleRequestDto;
+import io.perfume.api.sample.infrastructure.api.dto.SampleResponseDto;
+import io.perfume.api.sample.infrastructure.api.dto.UpdateSampleRequestDto;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Min;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'